num = 0
try:
    print(f'計算結果： {10/num}')
except:
    print('エラー')

